macros/AddonSidebar.ejs

```diff
@@ -12,145 +12,320 @@ function currentPageIsUnder(root) {
+    'WebExtensions/Comparison_with_the_Add-on_SDK': 'Comparaison avec le Add-on SDK',
+    'WebExtensions/Comparison_with_XUL_XPCOM_extensions': 'Comparaison avec les extensions XUL/XPCOM',
+    'WebExtensions/Chrome_incompatibilities': 'Incompatibilités avec Chrome',
+    'WebExtensions/Differences_between_desktop_and_Android': 'Différence entre le bureau et l\'Android',
+    'WebExtensions#Firefox_workflow': 'Déroulement avec Firefox',
+    'WebExtensions/User_experience_best_practices': 'Expérience utilisateur ?',
+    'WebExtensions/Temporary_Installation_in_Firefox': 'Installation temporaire dans Firefox',
+    'WebExtensions/Debugging': 'Débogage',
+    'WebExtensions/Testing_persistent_and_restart_features': 'Test des fonctionnalités persistantes et de redémarrage',
+    'WebExtensions/Developing_WebExtensions_for_Firefox_for_Android': 'Écrire des extensions pour Firefox sur Android',
+    'WebExtensions/Getting_started_with_web-ext': 'Démarrer avec web-ext',
+    'WebExtensions/web-ext_command_reference': 'Détail de la commande web-ext',
+    'WebExtensions/WebExtensions_and_the_Add-on_ID': 'Les WebExtensions et l\'ID Add-on',
+    'WebExtensions/Request_the_right_permissions': 'Demander les bonnes permissions',
+    'WebExtensions/API': 'Les APIs JavaScript',
+    'WebExtensions/manifest.json': 'Clés de Manifest',
+    'Themes': 'Thèmes',
+    'Themes/Theme_concepts': 'Thème de navigateurs',
+    'Themes/Theme_concepts_': 'Concepts des thèmes',
+    'Themes/Lightweight_themes': 'Thèmes légers',
+    'Themes/Lightweight_Themes/FAQ': 'FAQ des thèmes légers',
+    'Distribution': 'Publication et distribution',
+    'Distribution/Publishing': 'Publication de votre extension',
+    'Distribution/Signing': 'Signature et distribution de votre extension',
+    'WebExtensions/Package_your_extension_': 'Zipper votre extension',
+    'Distribution/Submitting_an_add-on': 'Soumettre une extension',
+    'Source_Code_Submission': 'Soumission du code source',
+    'Distribution/Resources_for_publishers': 'Ressources pour les développeurs',
+    'Listing': 'Créer une page attrayante pour votre extension',
+    'AMO/Policy/Reviews': 'Processus de revue',
+    'AMO/Policy/Agreement': 'Accord développeurs',
+    'AMO/Policy/Featured': 'Extensions mises en avant',
+    'Distribution/Retiring_your_extension': 'Retrait de votre extension',
+    'WebExtensions/Alternative_distribution_options': 'Distribuer votre module',
+    'WebExtensions/Alternative_distribution_options/Sideloading_add-ons': 'Pour installation manuelle',
+    'WebExtensions/Alternative_distribution_options/Add-ons_for_desktop_apps': 'Pour applications de bureau',
+    'WebExtensions/Alternative_distribution_options/Add-ons_in_the_enterprise': 'Pour une entreprise',
+    'Community_and_Support': 'Communauté et Support',
+    'Channels': 'Canaux de discussions',
+    'Blog': 'Blog extensions',
+    'Forums': 'Forum extensions',
+    '#Contact_us': 'Nous contacter'
+  }
+})
```

Comment:
End with a semicolon please.

Reply:
Let's blame the linter for that :slightly_smiling_face: fixed :+1: